README.md

```diff
@@ -106,6 +117,25 @@ mysql.query(SQL`SELECT * FROM `.append(mysql.escapeId(someUserInput)).append(SQL
 pg.query(SQL`SELECT * FROM `.append(pg.escapeIdentifier(someUserInput)).append(SQL` WHERE name = ${book} ORDER BY ${column} `).append(order))
 ```
 
+## Raw values with the RAW function / template tag
+Values returned from the RAW function (either by invocation on a string parameter or by use as a template tag) will be
+inserted raw when passed to the template literal or the `append()` function.
+
+```js
+const RAW = SQL.RAW
```

Comment:
I'd advocate
```js
const { raw, sql } = require('sql-template-strings')
```

Reply:
I prefer @felixfbecker's suggestion too.

Reply:
I'd suggest the default export be `sql`, so that both of the following options are possible (when `import`/`export` syntax makes it into node):

```js
import sql from 'sql-template-strings'
query(sql`select ... sql.raw('something'`)
```

```js
import { sql, raw } from 'sql-template-strings'
query(sql`select ... raw('something'`)
```